the-experts-BE/routes/getReview.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
+const router = express.Router({ mergeParams: true });
+const queries = require("../queries");
+
+router.get("/", function(req, res, next) {
+  queries.getReview(req.params.contractor_id, (err, response) => {
+    if (err) next(err);
+    res.send(response.rows);
```

Comment:
Use `res.json()` here to make it clear you are sending back data with `Content-Type: application/json`